lib/ldp.js

```diff
@@ -119,19 +119,24 @@ class LDP {
       })
   }
 
-  readContainerMeta (directory, callback) {
+  async readContainerMeta (directory, callback = function () {}) {
```

Comment:
Having both promise-based and callback-based resolving might be confusing. I propose to remove the callback parameter.